### ISSUE TITLE
[Feature Request] Return input when describing WE

### ISSUE THREAD

**Issue by lorensr** [temporalio/features#256]
### Is your feature request related to a problem? Please describe.

Hard to look up input based on WID. You need to do:

https://community.temporal.io/t/how-can-i-get-the-args-used-to-run-a-workflow-in-the-typsecript-sdk/7802/3?u=loren

plus payload decoding

### Describe the solution you'd like

`(await handle.describe()).input`, where input is array of decoded payloads

### Additional context

`describe()` could call both DescribeWorkflow and fetch history, or Server can add `input` to `DescribeWorkflowExecutionResponse`

https://api-docs.temporal.io/#temporal.api.workflowservice.v1.DescribeWorkflowExecutionResponse

**Comment by cretz** [comment]
> describe() could call both DescribeWorkflow and fetch history, or Server can add input to DescribeWorkflowExecutionResponse

I disagree with this issue. We offer users the ability to do describe and history fetch, we don't need to combine the calls for them. If it's too hard to call history and extract the input, we can maybe make that easier. But I don't think we should fetch history for everyone wanting to describe.

And I consider input as part of history, server should not change/hack just for someone not willing to make the proper history to get history input. This is not just workflow input, it's workflow result, signal input, etc. These are history.

> Hard to look up input based on WID

Can you provide proposals to make this easier SDK side without infecting the simple concept of "describe" and without unnecessary server alterations? We have the ability to get result, do you need a separate "get input"? Or do you need some helpers/shortcuts when getting history?
